docs/api/utility.rst

```diff
@@ -13,9 +13,9 @@ blt_assert_exists
 .. code-block:: cmake
 
     blt_assert_exists(
-      [DIRECTORY <path/to/directory/>]
-      [FILE <path/to/file>]
-      [TARGET <cmake-target]> )
+      [DIRECTORIES <dir1> [<dir2> ...] ]
+      [FILES <file1> [<file2> ...] ]
+      [TARGETS <target1> [<target2> ...] ] )
 
 Checks if the specified directory, file and/or cmake target exists and throws
```

Comment:
Minor: Please update this sentence to reflect that each of these can be plural.

Reply:
Fixed.